indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/payments/IndyPayments.kt

```diff
@@ -0,0 +1,19 @@
+package com.luxoft.blockchainlab.hyperledger.indy.payments
```

Comment:
Let's remove. Payment doesn't related to API re-design

Reply:
Oops..